pkg/scan/apk.go

```diff
@@ -87,7 +87,22 @@ func APK(f io.Reader) ([]*Finding, error) {
 
 	syftPkgs := packageCollection.Sorted()
 
-	datastore, _, dbCloser, err := grype.LoadVulnerabilityDB(grypeDBConfig, true)
+	updateDB := true
+	if localDBFile != "" {
+		fmt.Printf("Loading local grype DB %s...\n", localDBFile)
```

Comment:
same comment re: putting progress updates like this onto stderr

Reply:
done